### ISSUE TITLE
More barrels and boxes.

### ISSUE THREAD

**Issue by igorko** [igorko/flare-mod-noname#52]
It's a little boring when you don't find anything on your way.

**Comment by sang-froid** [comment]
Yes I agree with you I also prefer to find often chests / barrels even if they are empty or uninteresting
